go/vt/vttablet/tabletserver/vstreamer/rowstreamer.go

```diff
@@ -297,6 +310,8 @@ func (rs *rowStreamer) streamQuery(conn *snapshotConn, send func(*binlogdatapb.V
 		if err != nil {
 			return err
 		}
+	} else {
+		send(response)
```

Comment:
you want to wrap this in error checking since otherwise you'll just ignore it  and proceed

Reply:
ended up refactoring to just use one `send(response)` and error check. Should be all set now